geometry_msgs/Pose wanted_pos
---
string status
